### ISSUE TITLE
AVT 3 Dropdown - VO does not announce the option after it is selected or the options position in the list..

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4373]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React

## Detailed Description
Go to the React Component 
Start VO on macOS
1. Tab to the Dropdown, VO announces that it Open Menu pop-up 1 of 1.
**Expected Result** - VO announces the label "Option 1" and that it is a pop-up button.
2. Use the Enter key to open the dropdown and move to the next option. VO announces the option, but does not announce the position (i.e. 1 of 4, 2 of 4, etc).
**Expected Result**  - VO announces the Option # and position. 
3. Tab away from the dropdown and then tab back. VO announces, "Open Menu pop-up".
**Expected Result**- VO announces the that it is a menu pop-up and the current selection. 

See [ARIA Authoring Practices Dropdown Listbox example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html).

**Comment by asudoh** [comment]
Fixing 2 (#5193). Wrt 1. and 3., our Dropdown has a state where no selection has been made (yet), instead of the first item automatically selected as the issue author may have expected - Thus keeping it intact (just letting screen reader user know that it's a pop-up button).
